capitalone/rubicon
# Add tests and additional logger functionality

## What
  * Cleanup and add more context to the sklearn integration notebook
  * Unit test the logger classes
  * Ensure that the logger is used to log metrics in the RubiconPipeline (previously was off the experiment directly)

## How to Test
  * Run thru updated notebook, check the unit tests

I'm debating unit test or integration test for the RubiconPipeline functionality
